nixos/modules/services/misc/paperless.nix

```diff
@@ -439,5 +489,37 @@ in
         gid = config.ids.gids.paperless;
       };
     };
-  };
+  }) (lib.mkIf (cfg.enable && cfg.exporter.enable) {
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.exporter.directory}' - ${cfg.user} ${config.users.users.${cfg.user}.group} - -"
+    ];
+
+    systemd.timers.paperless-exporter = lib.mkIf (cfg.exporter.onCalendar != null) {
+      description = "paperless exporter timer";
+      wantedBy = [ "timers.target" ];
+      timerConfig = {
+        OnCalendar = cfg.exporter.onCalendar;
+        AccuracySec = "5m";
+      };
+    };
```

Comment:
This could perhaps be simplified by just setting the `systemd.service.<name>.onCalendar` option instead.

Reply:
As in "replacing the whole expression and dropping everything beside the OnCalendar option"? I'll try that.

Reply:
It appears we have to keep the `wantedBy` setting. So I can simplify it a bit but not much.

I thought about using `startAt` on the service instead, however, that uses `[]` as the setting to signal "no timer" and would make it harder/less obvious how to disable the timer ...

Reply:
I meant the abstraction we have for `systemd.service`. If all you need is run the service on a schedule, there's a handy abstraction which sets that up correctly; no need to declare a timer separately.

Reply:
Sorry, I misremembered the name. It's `startAt`. See i.e. https://github.com/Atemu/nixos-config/blob/5323b915a63748c7ffa891f813a753fa16601df5/modules/paperless/module.nix#L94.

Reply:
Yeah, I noticed that abstraction, but was a bit unhappy that you'd have to override with `[]` instead of `null` if you want to disable it later ... thoughts?

Reply:
You'd use mkIf with a custom condition anyways.